src/scanoss/scanners/scanner_hfh.py

```
@@ -161,7 +163,50 @@ def _format_plain_output(self) -> str:
         )
 
     def _format_cyclonedx_output(self) -> str:
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)